src/pages/configurePatch/configurePatchCore/index.tsx

```diff
@@ -34,9 +34,9 @@ import {
   VariantTasksState,
   useConfigurePatch,
 } from "hooks/useConfigurePatch";
-import { ParametersContent } from "pages/configurePatch/ParametersContent";
-import { ConfigureBuildVariants } from "./configurePatchCore/ConfigureBuildVariants";
-import { ConfigureTasks } from "./configurePatchCore/ConfigureTasks";
+import { ConfigureBuildVariants } from "./ConfigureBuildVariants";
+import { ConfigureTasks } from "./ConfigureTasks";
+import { ParametersContent } from "./ParametersContent";
 
 interface Props {
```

Comment:
hmm I guess the compile errors are coming from not picking up the change from `configurePatchCore` => `ConfigurePatchCore`?

I might recommend renaming the folder to something entirely different, or removing it completely -- not sure that you need the extra level of nesting here

Reply:
So strange how this didn't get picked up while developing.